salsa_dancing_molecules/unit_test/test_capacity.py

```diff
@@ -25,7 +26,7 @@ def test_capacity_NVE():
     init_mock()
 
     # Mock funtions
-    expected = -1.1518362268490951e-12  # Calculated expected value
+    expected = -3.704188769270802e-05  # Calculated expected value
```

Comment:
Are these values calculated from some other source or are they just taken from what the function spits out? If they come straight from the function, I don't know if we can say they validate it works correctly.